src/interface/src/app/treatments/map-base-layer/map-base-layer.component.html

```diff
@@ -0,0 +1,15 @@
+<details open>
```

Comment:
this could become its own component (the styles are going to be somewhat repeated for the other map layer configs).
Will probably do a component for the next one

Reply:
ok. sounds reasonable